src/Middleware/Spa/SpaServices.Extensions/src/Proxying/SpaProxy.cs

```diff
@@ -68,9 +68,10 @@ public static HttpClient CreateHttpClientForProxy(TimeSpan requestTimeout)
         // when proxying to Angular CLI middleware: we won't know what port it's listening
         // on until it finishes starting up.
         var baseUri = await baseUriTask;
-        var targetUri = new Uri(
-            baseUri,
-            context.Request.Path + context.Request.QueryString);
+        var targetUri = new Uri(baseUri,
+			baseUri.AbsoluteUri
+			+ (baseUri.AbsoluteUri.EndsWith("/", StringComparison.OrdinalIgnoreCase) ? context.Request.Path.ToString().Substring(1) : context.Request.Path)
```

Comment:
What happens if `context.Request.Path` is empty? Can we add a test case for that?

Would it work to just trim the trailing "/" characters off the end of `baseUri.AbsoluteUri`, since `context.Request.Path` is [guaranteed](https://github.com/dotnet/aspnetcore/blob/8eadd8f7e2b46387ea6f121a8d9f9bf003cf3f2c/src/Http/Http.Abstractions/src/PathString.cs#L35) to start with a `/` if it's non-empty?

Reply:
Also, can we use the `Uri` constructor overload that takes a single URI string? It looks like we're treating the `relativeUri` parameter as the base URI anyway.

Reply:
@MackinnonBuck Good catch! I have made these changes, thanks for the feedback.